talk/morelanguage/constexpr.tex

```diff
@@ -75,20 +75,20 @@
 \begin{frame}[fragile]
   \frametitlecpp[11]{Real life example}
   \begin{cppcode*}{}
-    constexpr float toSI(const float v, const char unit) {
+    constexpr float toSI(float v, char unit) {
```

Comment:
Do we have a style recommendation for making by-value parameters `const`? It's not a bad thing per se, but adds some noise to the slides.